### PR TITLE
styled the info bar

### PR DESCRIPTION
**Pull request by garvpatidar04**
Please review my committed pull request and tell me if there is any mistake or improvement I can make to this issue.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (codingwithchad) [inline .vscode/settings.json:2]: Please add .vscode/settings to .gitignore. as these settings are for you only.

Comment (garvpatidar04) [reply .vscode/settings.json:2]: Ok, I will do that.

Comment (garvpatidar04) [reply .vscode/settings.json]: I have deleted the folder.

Comment (codingwithchad) [reply .vscode/settings.json]: Thank you for working on this, but I am going to handle it.

**Pull request closed without merging**